src/sync/installation.ts

```diff
@@ -122,7 +122,7 @@ const updateJobStatus = async (
 		return;
 	}
 
-	const status = edges.length > 0 ? "pending" : "complete";
```

Comment:
`edges` was `null` in some cases. This would throw an error that is not caught anywhere, which means no follow-up job is queued and the sync is effectively "stalled". After 15 minutes of no updates, when the installation is loaded from the database again, the sync status will be set to `FAILED` due to inactivity.